apps/tangle-cloud/src/pages/blueprints/[id]/deploy/DeploySteps/components/OperatorTable.tsx

```diff
@@ -0,0 +1,304 @@
+import useNetworkStore from '@tangle-network/tangle-shared-ui/context/useNetworkStore';
+import useOperatorsServices from '@tangle-network/tangle-shared-ui/data/blueprints/useOperatorsServices';
+import useRestakeOperatorMap from '@tangle-network/tangle-shared-ui/data/restake/useRestakeOperatorMap';
+import useIdentities from '@tangle-network/tangle-shared-ui/hooks/useIdentities';
+import {
+  assertSubstrateAddress,
+  Avatar,
+  CheckBox,
+  EMPTY_VALUE_PLACEHOLDER,
+  EnergyChipColors,
+  EnergyChipStack,
+  ExternalLinkIcon,
+  fuzzyFilter,
+  getRoundedAmountString,
+  KeyValueWithButton,
+  Table,
+  Typography,
+} from '@tangle-network/ui-components';
+import { FC, useMemo } from 'react';
+import { OperatorSelectionTable } from '../type';
+import safeFormatUnits from '@tangle-network/tangle-shared-ui/utils/safeFormatUnits';
+import useRestakeAssets from '@tangle-network/tangle-shared-ui/data/restake/useRestakeAssets';
+import delegationsToVaultTokens from '@tangle-network/tangle-shared-ui/utils/restake/delegationsToVaultTokens';
+import {
+  createColumnHelper,
+  getFilteredRowModel,
+  getSortedRowModel,
+  getCoreRowModel,
+  useReactTable,
+  getPaginationRowModel,
+  TableOptions,
+} from '@tanstack/react-table';
+import { sortByAddressOrIdentity } from '@tangle-network/tangle-shared-ui/components/tables/utils';
+import TableCellWrapper from '@tangle-network/tangle-shared-ui/components/tables/TableCellWrapper';
+import VaultsDropdown from '@tangle-network/tangle-shared-ui/components/tables/Operators/VaultsDropdown';
+import { TableVariant } from '@tangle-network/ui-components/components/Table/types';
+import useBlueprintRegisteredOperator from '@tangle-network/tangle-shared-ui/data/blueprints/useBlueprintRegisteredOperator';
+import { useParams } from 'react-router';
+import { getOperatorPricing } from '../../../../../../utils';
+
+const COLUMN_HELPER = createColumnHelper<OperatorSelectionTable>();
+
+type Props = Omit<
+  TableOptions<OperatorSelectionTable>,
+  'data' | 'columns' | 'getCoreRowModel'
+> & {
+  advanceFilter?: (row: OperatorSelectionTable) => boolean;
+};
+
+export const OperatorTable: FC<Props> = ({ advanceFilter, ...tableProps }) => {
+  const { id: _blueprintId } = useParams();
+  const blueprintId = Number(_blueprintId);
+
+  const { assets } = useRestakeAssets();
+
+  const { operatorMap: restakeOperatorMap } = useRestakeOperatorMap();
+
+  const { result: _registeredOperators } =
```

Comment:
To avoid ignoring Prettier/ESLint warnings:

```suggestion
  const { result: registeredOperators_ } =
```